docs/notebooks/2.3 Dust Soiling with Kimber.ipynb

```diff
@@ -0,0 +1,398 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "43cd9359-32f3-455a-b9a2-3f419adacc32",
+   "metadata": {},
+   "source": [
+    "July 2024 has been an exceptional month in terms of irradiance anomalies in North America. In one of [Solcast's weekly PVMag publications](https://www.pv-magazine.com/2024/08/02/aerosols-from-wildfires-slash-north-american-solar-production/) we have highlighted how irradiance was severely impacted in the north west of  the continent by aerosols deriving from the raging Canadian wildfires.\n",
+    "\n",
+    "In addition to the reduced irradiance from the aerosol's absorption, another factor to take into account into the overall impact of these aerosols on solar generation is dust soilig as the accumulated particles block or scatter incident light, further reducing power output.g.\n",
```

Comment:
Typo "soilig"
Typo "output.g"
For the physical process of soiling on a panel, is scattering involved?  I would have thought the dust is either reflecting or absorbing the incident light.  Could just say "block".